view/main_content_panel.py

```diff
@@ -2490,7 +2490,7 @@ def _do_build_database_thread(self) -> None:
 
     def _do_edit_steam_webapi_key(self) -> None:
         """
-        Opens a QDialogInput that allows the user to edit their Steam apikey
+        Opens a QDialogInput that allows the user to edit their Steam api key
```

Comment:
Technically, this should be "API-key" :stuck_out_tongue:

Reply:
done